app/controllers/batch_connect/session_contexts_controller.rb

```diff
@@ -33,7 +33,7 @@ def create
     respond_to do |format|
       if @session.save(app: @app, context: @session_context, format: @render_format)
         cache_file.write(@session_context.to_json)  # save context to cache file
-        format.html { redirect_to batch_connect_sessions_url, notice: 'Session was successfully created.' }
+        format.html { redirect_to batch_connect_sessions_url, notice: t('dashboard.bc.sessions.status_blurb.create_success') }
```

Comment:
Instead of bc we should use the full batch_connect

```
dashboard.batch_connect.sessions.status_blurb.create_success
```

This is so that we follow similar conventions where we use batch connect in other locations i.e. https://osc.github.io/ood-documentation/master/installation/cluster-config-schema.html?highlight=batch_connect